public/slack-app-registry.json

```diff
@@ -13,11 +13,16 @@
                 "description" : "Returns a random xkcd comic.",
                 "manual" : "THERE IS NO DOCUMENTATION WEBSITE. MAKE YOUR OWN."                
             },
-	    "lowPriority" : {
-		"usage" : "/struggle lowPriority",
-		"description" : "Mark has no idea what this function does.",
-		"manual" : "Hope Brett doesn't kill us."
-	    }
+            "lowPriority" : {
+                "usage" : "/struggle lowPriority",
+                "description" : "Mark has no idea what this function does.",
+                "manual" : "Hope Brett doesn't kill us."
+            },
+            "inviteme" : {
+                "usage" : "/struggle dev invite",
+                "description" : "Returns a link that will send automated invite to jdsb-wrecking-ball workspace",
+                "manual" : "Just type `/struggle dev invite` in slack channel"
```

Comment:
Go ahead and add an "author" entry. You can put your name or alias in there e.g.

"author" : "lisakeem"